ui/src/pages/frontsite/Documents/DocumentsDetails/LoanRequestForm/state/actions.js

```diff
@@ -18,24 +13,34 @@ export const requestLoanForDocument = (
       type: IS_LOADING,
     });
     const today = toShortDate(DateTime.local());
+
     try {
-      await loanApi.doRequest(documentPid, sessionManager.user.id, {
-        requestExpireDate: requestEndDate,
-        requestStartDate: today,
-        deliveryMethod: deliveryMethod,
-      });
-      dispatch(
-        sendSuccessNotification(
-          'Success!',
-          `You have requested to loan this book.`
-        )
+      const response = await loanApi.doRequest(
+        documentPid,
+        sessionManager.user.id,
+        {
+          requestExpireDate: requestEndDate,
+          requestStartDate: today,
+          deliveryMethod: deliveryMethod,
+        }
       );
+      dispatch({
```

Comment:
@KonstantinaStoikou @kprzerwa for the next PR, please make sure that you add her, just before the `dispatch`, the `await delay()` to allow ES to index.